src/types/zenvia.ts

```diff
@@ -163,3 +163,18 @@ export interface IWebhook {
     [header: string]: string;
   };
 }
+
+export interface IChannels {
```

Comment:
I know there is already a IChannel, but I am not so sure about simply putting this in plural is the best idea. Do we have any better alternative?

Reply:
What you guys think? @wmangelardo @hmagarotto @rodrigokamada